src/components/Analysis/Analyses/index.tsx

```diff
@@ -35,7 +34,8 @@ const AnalysesTable: React.FC<AssociatedAnaysesProps> = ({ rootEndpoint }) => {
       page_size: analysesPageSize,
     }
   );
-  if (loading && !isStale) return <Loading size="small" />;
```

Comment:
Dropping the `!isStale` check means that a change in the page number will return the loading spinner now, instead of the flashing stale state of the table, right?